cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/ProgramSchedule.java

```diff
@@ -43,6 +45,7 @@ public ProgramSchedule(String name, String description,
     this.name = name;
```

Comment:
But we can have the `getName()` method returns `scheduleId.getSchedule()`, right?